frontend/src/app/forms/auth-guard.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @description
+ *   This page handles the basic operation of the Guard.
+ *
+ */
+
+import { Injectable } from '@angular/core';
+import {
+  ActivatedRouteSnapshot,
+  CanDeactivate,
+  RouterStateSnapshot,
+} from '@angular/router';
+import { Observable } from 'rxjs';
+
+import { FormsPage } from './forms.page';
+import { FULL_IMAGE_VIEWER_PATH } from './router-constants';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthGuard implements CanDeactivate<FormsPage> {
+  /**
+   * CanDeactivate method of the Guard to pretend going back to home page accidently without a warning
```

Comment:
There is a typo: "pretend" instead of "prevent". And "accidently" instead of "accidentally".

Also, I would prefer a wording like "... going back to the previous page ..." since it is not necessarily the home page the user might go back to.